pkg/solana/chain.go

```diff
@@ -232,7 +233,38 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 	return &ch, nil
 }
 
-// ChainService interface
+func (c *chain) LatestHead(_ context.Context) (relaytypes.Head, error) {
+	sc, err := c.getClient()
+	if err != nil {
+		return types.Head{}, err
```

Comment:
We have two times the same import: 
```
"github.com/smartcontractkit/chainlink-common/pkg/types"
relaytypes "github.com/smartcontractkit/chainlink-common/pkg/types"
```

That creates confusion when returning `types.Head{}` in some cases and `relaytypes.Head{}` in others

Reply:
weird how linter allowed this